### DISCUSSION TITLE
Mrx 

Giant script✅

### DISCUSSION THREAD

**Discussion by 1xLucas** [community/community#138137] [Mobile] [answered]
local player = game.Players.LocalPlayer
local character = player.Character or player.CharacterAdded:Wait()

local function makeGiant()
    local scaleMultiplier = 5
    local humanoid = character:WaitForChild("Humanoid")
    
    for _, part in pairs(character:GetChildren()) do
        if part:IsA("BasePart") then
            part.Size = part.Size * scaleMultiplier
        elseif part:IsA("Accessory") then
            if part:FindFirstChild("Handle") then
                part.Handle.Size = part.Handle.Size * scaleMultiplier
            end
        end
    end
    
    humanoid.HipHeight = humanoid.HipHeight * scaleMultiplier
    humanoid.BodyDepthScale.Value = scaleMultiplier
    humanoid.BodyHeightScale.Value = scaleMultiplier
    humanoid.BodyWidthScale.Value = scaleMultiplier
end

makeGiant()

**Comment by ebndev** [comment]
Hi @Mrxscript1, thanks for being a part of the GitHub Community!

Can you provide more details about what you are sharing? i.e if you are seeking help or which cases someone may be able to use your script. Otherwise, we may close this issue to keep the space tidy 😄

**Reply by 1xLucas** [reply, marked as answer]
This is the script that you can use in the Roblox game, of course, the bad part is that the script gave an error, but new scripts will come, don't miss it.

**Reply by sharie90** [reply]
Aku juga mau

**Comment by Iron-Ham** [comment]
Hi @Mrxscript1 – I'm going to close this discussion out, as the `Mobile` tag is meant for assistance/feedback with the GitHub Mobile apps for iOS and Android.

**Comment by Aseem007-007** [comment]
ي
